src/Transpiler.ts

```diff
@@ -1673,4 +1690,17 @@ export abstract class LuaTranspiler {
 
         return result;
     }
+
+    public checkForLuaLibType(type: ts.Type): void {
```

Comment:
The naming of this function seems a bit of to me. As it checks for the time, but also adds the feature.